backend-services/.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
We could also ignore the python cache files.

I had a look at JP's bookmark initial implementation PR and we could use the same .gitignore file he used - https://github.com/ntls-io/nautilus-wallet/pull/432/files#diff-d24881b3a26df5602698d4ee0a6b60bb252e4b4836a53d2f977fc7ab50f74ee8